packages/@divvi/mobile/src/public/types.tsx

```diff
@@ -239,6 +239,8 @@ export interface PublicAppConfig<tabScreenConfigs extends TabScreenConfig[] = Ta
         }
       }
     }
+    disableNfts?: boolean
+    hideCashInTokenFilters?: boolean
```

Comment:
we also want to hide swap token filters and import token feature, but those are configurable via statsig, so just included these 2 here, but we can also allow configuring those here to keep them consistent. Thoughts?

Reply:
We were chatting about something similar to this with @kathaypacific.
We probably want configurable defaults for statsig for apps that don't want to use statsig.
So likely we'll add what you said here.

Reply:
kk in that case I'll hold for now and use statsig for cKash (we already have an account).